lib/connection.js

```diff
@@ -229,6 +253,21 @@ export default function Connection(options = {}) {
     return ready && ended && ended()
   }
 
+  function oncopy() {
+    backend.query.writable.push = ({ chunk, error, callback }) => {
+      error
```

Comment:
Risk of unhandled promise rejection
```suggestion
      rerturn error
```

Reply:
I'm not sure about that one since it's actually sent to the socket:
```ts
    backend.query.writable.push = ({ chunk, error, callback }) => {
      error
          ? socket.write(frontend.CopyFail(error))
          : chunk === null
              ? socket.write(frontend.CopyDone())
              : socket.write(frontend.CopyData(chunk), callback)
    }